### PR TITLE
Paddle support for classic_solvers.py

### PR DESCRIPTION
**Pull request by Arist12**

### CODE REVIEW

File: pygmtools/paddle_backend.py [hunk 1]
```diff
@@ -0,0 +1,463 @@
+import itertools
+from matplotlib.pyplot import axis
```
Comment (rogerwwww) [inline pygmtools/paddle_backend.py:2]: Is matplotlib used in this script?

File: tests/test_multi_graph_solvers.py [hunk 2]
```diff
@@ -1,187 +0,0 @@
-import sys
```
Comment (rogerwwww) [inline tests/test_multi_graph_solvers.py:1]: This file should not be deleted

Comment (Arist12) [reply tests/test_multi_graph_solvers.py:1]: I delete this file locally so that I can speed up my test.

[the remark this answers is not in the document]
Comment (Arist12) [reply tests/test_multi_graph_solvers.py:1]: ok, I've restored it!

**Comment by Arist12** [comment]
the final version has been pushed with comments removed and multigraph tester restored.

**Comment by 1115712590** [comment]
Nice work!

**Pull request merged**